cli/test/serve.spec.js

```diff
@@ -16,6 +16,7 @@ const serve_args = [
   '--port=0',
   '--auto-create-collection',
   `--project-name=${project_name}`,
+  `--token-secret=test-token`
```

Comment:
I have taken the easy route of adding the token-secret as an argument here. Is this okay?